crates/pedersen/src/lib.rs

```diff
@@ -1,477 +1,6 @@
 pub mod curve;
+pub mod field;
+pub mod hash;
 
```

Comment:
The previous API should most likely just be re-exported at this lib level to avoid rehashing the `use`'s from other parts of the repo. Meaning something like:

```suggestion
pub use hash::{StarkHash, pedersen_hash_preprocessed as pedersen_hash};
```

Did not check this suggestion so it might not be enough. Something to consider however is that the name `pedersen_hash` does become a bit ambigious...